### PR TITLE
WIP add scripts for load testing

### PR DESCRIPTION
**Pull request by vrutkovs**
cc @steveeJ

### CODE REVIEW

File: hack/load-testing.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    echo "Testing workers ${workers}, rate ${rate} -> ${file}"
+    sed "s,GRAPH_URL,${GRAPH_URL},g" vegeta.targets | \
+      vegeta attack -format http -workers=${workers} -rate=${rate} -duration ${duration} > ${file}
+    vegeta report -type=text ${file}
+  done
+done
+
+vegeta report -type='hist[0,50ms,100ms,500ms,1s,5s,10s]' /tmp/results/*.bin
+
+sleep infinity
```
Comment (steveej) [inline hack/load-testing.sh:27]: Why can't it exit?

Comment (vrutkovs) [reply hack/load-testing.sh:27]: I've added to inspect pod output (this script was running in a deployment). The final version won't have this

**Review by LalatenduMohanty — CHANGES_REQUESTED** [review]
I would suggest a directory named `load-test` inside `hack` and all the files should be moved there.

**Comment by vrutkovs** [comment]
>I would suggest a directory named load-test inside hack and all the files should be moved there.

We already have `dist` for "public" files and `hack` for internal scripts (at least as I understood it). Load testing probably would be used by stage/prod scripts - why would need a new dir for that?

**Comment by vrutkovs** [comment]
Also this is superseded by https://github.com/openshift/cincinnati/pull/249

**Pull request closed without merging**